updater/site/github.py

```diff
@@ -0,0 +1,39 @@
+import requests
+import re
+import random
+from updater.site.abstract_site import AbstractSite
+
+
+class Github(AbstractSite):
+    _URL = 'https://github.com/'
+
+    def __init__(self, url: str):
+        if '/tree/master' not in url:
+            url = (url + '/tree/master')
+        super().__init__(url)
+
+    @classmethod
+    def get_supported_urls(cls):
+        return [cls._URL]
+
+    def find_zip_url(self):
+        return self.url.replace('/tree/', '/archive/', 1) + '.zip'
+
+    def get_latest_version(self):
+        try:
+            print(self.url)
+            response = requests.get(self.url)
+            response.raise_for_status()
+            content = str(response.content)
+            version = re.search(
+                r'<a data-pjax.*?\/commit\/(?P<hash>.*?)">',
+                content).group('hash')
+            return version.strip()
```

Comment:
Does this `strip()` still need to be used?

Reply:
No you are right, just me being careless. Removed it.